contributors/design-proposals/cpu-manager.md

```diff
@@ -158,6 +163,26 @@ node. Once allocated at pod admission time, an exclusive CPU remains
 assigned to a single container for the lifetime of the pod (until it
 becomes terminal.)
 
+##### Configuration
+
+Operators can set the number of CPUs that pods may run on through a new
+Kubelet configuration setting `--cpu-manager-static-num-cpus`, which
```

Comment:
`--cpu-manager-static-num-cpus` suggests that it's valid only for **static** policy - it that the case ?